### DISCUSSION TITLE
Baro needs more animation.

### DISCUSSION THREAD

**Discussion by Rehexr** [FakeFishGames/Barotrauma#10482] [Ideas and suggestions]
Honestly, I feel this was way overdue a while ago, and it makes me desperately wish for it. So I'll just start from the beginning; Baro could really use some new animations.

This would account for a couple of things:

**Idle animations**
Idle animations, as a very simple thing, could benefit the livelyness of both outposts and players alike, negating a lot of the roboticness that the characters face.In outposts, bots frequently just stand around, staring into nothingness; something as simple as some animations of them scratching themselves, messing around with something in their hands, looking around, or shifting their clothes could do a lot to negate that.
Same goes for players, as they just stand coldly when not moving.

**Emoting animations**
Characters currently do not emote at all, with all events being mostly purely text-based and outpost managers just staring at you as they hand out missions. To be honest, just some animations with them moving their arms to indicate that they were speaking would suffice.
Either that or have security officers or VIPs walk with their hands behind their backs and heads held high as if they own the place; either way, it would do a lot to characterize them and give them some more uniqueness.
That, or having limb-specific animations, such as holding your head or leg if it has a lot of damage, easily overridable if you need to use an item, and a significant improvement over just limping if your leg is injured.
In general, it would allow characters to emote with body language rather than emotion because face animations are most likely a limitation of how it's set up.

**Looking and head tilting**
Frankly, I'm almost surprised this isn't much of a thing yet, since the head tilting while using items was included. Simply being able to tilt your head without an item or while holding right click regardless of an item in your hand could help players emote a little more simply by visually looking up or down, even if only for roleplay purposes.

**More movement animations**
Simple movement animations would be beneficial, as they haven't received much attention since Legacy, and things like standing up, etc. still happen instinctively, making characters look a little like cartoon characters completely running on campaign, just jumping up.
Or even just a single stopping step to indicate the player turning around—to be honest, it would do just as much to make characters seem less like husks to be controlled by the player.

**Characters interacting with each other**
Another mostly bot-related one is that characters could interact with one another, such as outpost citizens giving another an encouraging pat on the back or an event character shaking the players' hands when meeting them. There's not much to say here, though, as a lot of uses for this would be situational.
Although something like medics padding someone down to check for injuries rather than just holding them by the hips would be a nice addition.

**Interaction animations**
Last but not least, the game could use some animations to indicate interactions with sub-items, especially containers and nav terminals. Just opening a container door or pressing some buttons on a nav terminal could do a lot to make the characters look busier than just standing nearby, not doing much at all.
Although there are some concerns with this, such as custom-scaled items having issues with this, I could still see it being a toggleable option for selection during sub editing, with vanilla subs already being in an optimal condition to use this.I feel like the ability to sit in a chair and interact with terminals was already a step in the right direction, but I would love to see more.

**In conclusion, I guess**
I understand that a lot of this is limited by how the animations are currently set up; making things dynamically animate without removing player control isn't exactly a small request, but to be honest, this is really just the next step in how to improve the game on a level.
While new features and faction updates all evolve the game's gameplay, the animations that accompany them have been given no love for a while now, sadly.
